src/Microsoft.Extensions.FileProviders.Abstractions/NoopChangeToken.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Extensions.FileProviders
 {
-    internal class NoopChangeToken : IChangeToken
+    public class NoopChangeToken : IChangeToken
```

Comment:
Naming concern since we're making this type public - should this be called `NullChangeToken`? We name other things similarly e.g. `NullLogger` for things that don't do much. Thoughts @Eilon ?